scripts/output_env.sh

```diff
@@ -41,81 +45,100 @@ print_version()
         VERSION_STR=`echo "$VERSION_STR" | $FILTER`
     done
 
-    echo "* ${BIN##*/}: $BIN: $VERSION_STR"
+    echo " * ${BIN##*/}$VARIANT: ${BIN} : ${VERSION_STR} "
 }
 
-print_version "uname" "-a" ""
+echo "Platform:\n"
```

Comment:
Whether `\n` prints a newline or backslash-n depends on the shell. For portability, use
```
echo "Platform:"
echo
```
or
```
printf 'Platform:\n\n'
```
Why do you want an empty line here anyway?

Reply:
The empty line was because I was trying to give a sense of section headers. I've refined this  so hopefully it's more apparent. I've also removed the `"\n"` from all `echo`s.